examples-standalone/coffee-warehouse/src/app/app.component.html

```diff
@@ -0,0 +1,7 @@
+<header-component (toggle)="toggleDrawer(drawer)" [selectedPage]="selected"></header-component>
+<kendo-drawer-container>
```

Comment:
I would order the bindings a bit - each on a separate line when there is more than 1